src/main/java/com/jcabi/maven/plugin/AjcMojo.java

```diff
@@ -106,6 +106,17 @@
     private transient File classesDirectory;
 
     /**
+     * Directory in which uwoven classes are copied.
+     * @checkstyle MemberNameCheck (10 lines)
+     */
+    @Parameter(
+        required = false,
+        readonly = false,
+        defaultValue = "${project.build.outputDirectory}"
```

Comment:
@amihaiemil thanks, makes sense. We just need to document your thoughts on page.